pkg/acquisition/file_reader.go

```diff
@@ -97,13 +97,16 @@ func LoadAcquisCtxConfigFile(config *csconfig.CrowdsecServiceCfg) ([]FileCtx, er
 		err = dec.Decode(&t)
 		if err != nil {
 			if err == io.EOF {
-				log.Tracef("End of yaml file")
+				log.Printf("End of yaml file")
```

Comment:
I'm not sure we want this one in info as we're going to hit this case every time ?